scripts/build.sh

```diff
@@ -11,22 +11,24 @@ cd "${CONSTRUCT_ROOT}"
 
 echo "***** Install constructor *****"
 
+CF_CHANNEL_NAME="${CF_CHANNEL_NAME:-conda-forge}"
```

Comment:
```suggestion
MINIFORGE_CHANNEL_NAME="${MINIFORGE_CHANNEL_NAME:-conda-forge}"
```

right? you want to be able to change the main channel is that correct?

can you please explain why you need this? this is to install constructor and the related dependencies, it has nothing to do with the final output.

Reply:
Correct. We have internal channels that are cached from proxies pulled from conda-forge.

Reply:
We are restricted from pulling from conda-forge directly so we need to use our proxy caches. We wanted to use the same source without modifying after a release was published.